src/invstorage.cc

```diff
@@ -52,9 +52,63 @@ void INvStorage::importData(const ImportConfig& import_config) {
 boost::shared_ptr<INvStorage> INvStorage::newStorage(const StorageConfig& config) {
   switch (config.type) {
     case kSqlite:
+      if (!boost::filesystem::exists(config.sqldb_path)) {
+        boost::shared_ptr<INvStorage> sql_storage(new SQLStorage(config));
+        boost::shared_ptr<INvStorage> fs_storage(new FSStorage(config));
```

Comment:
That shouldn't work to my understanding, because the new `[Storage]` configuration will be tuned for SQL storage. What we should I believe is to hardcode a config snippet for FSStorage to match what we have had before normally.

Reply:
But how could we know the old configuration if it wasn't default?

Reply:
We could use libostree to get previous deployment and extract configuration from there. But then there is a different set of open issues (what happens if we change sota.toml location between updates for example), and I believe using the default one should work for all our users.

Reply:
I just added separate commit, please look if did it right.

Reply:
@patriotyk The FSStorage should be created with old_config, not SQLStorage. Might be the reason for segfault in tests as well.

Reply:
Oh, yes it's just a mistake

Reply:
I changed it  to work in another way. There was a problem with tests because in tests I need to setup "path" property. Is it ok? Or maybe You plan to replace sqldb_path with path in the future?

Reply:
@patriotyk Yes, sqldb_path will be merged with path in the end. I liked the previous solution, the only thing that had to be fixed there is we need a way to override the path of old file system storage for tests. I believe it can be done either with build options (see e.g. TEST_PKCS11_MODULE_PATH) of with additional parameters to INvStorage functions.